### PR TITLE
Change variable names in config.toml to snake case

### LINKED ISSUE #87: Migrate config.toml to snake_case
**Is this a BUG REPORT or FEATURE REQUEST?**:
feature

**What happened**:
We decided to use snake_case per recommendation from Hugo.

**What you expected to happen**:
Use snake_case in config.toml.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:
Even so Hugo doesn't support camelCase well and recommends snake_case in config.toml they actually use camelCase :man_facepalming: At least we should be consistent and also use camel_case in config.toml.

### PR DESCRIPTION
**Pull request by mpourismaiel**
**What this PR does / why we need it**:
Change config.toml to snake_case

**Which issue this PR fixes** *(optional - uncomment and add issue)*:
fixes #87

### CODE REVIEW

**Comment by stp-ip** [comment]
Let's do the fix, after we merge #108 and with that the update to Hugo 0.40.3.

**Comment by mpourismaiel** [comment]
@stp-ip Is there any convention Hugo keeps true? googleAnalytics is lower camelCase and LanguageCode is upper camelCase while Hugo claims snake_case is preferred. Anyways, could you please check it again?

**Comment by stp-ip** [comment]
Seems like in that regard hugo is horrible. Seems good will test it and merge.

File: exampleSite/config.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Site settings
 baseurl = "/"
-languageCode = "en-us"
+LanguageCode = "en-us"
```
Comment (stp-ip) [inline exampleSite/config.toml]: It can be lowercase here, but needs to be uppercase in the html layouts afaik.

File: exampleSite/config.toml [hunk 2]
```diff
@@ -15,26 +15,26 @@ googleAnalytics = ""
...
-    textNavigation = "Syna"
+    logo_navigation = "logo.svg"
+    text_navigation = "Syna"
 
   [params.style]
     #background = "secondary"
-    #backgroundNavigation = "light" # white, light, secondary, primary, dark - default: dark
-    #backgroundFooter = "light" # white, light, secondary, primary, dark - default: dark
+    #background_navigation = "light" # white, light, secondary, primary, dark - default: dark
+    #background_footer = "light" # white, light, secondary, primary, dark - default: dark
```
Comment (stp-ip) [inline exampleSite/config.toml]: background_footer isn't used anymore correct? As we moved the copyright into it a fragment.

File: layouts/partials/fragments/copyright.html [hunk 3]
```diff
@@ -48,11 +48,11 @@
           </div>
         </div>
       {{ end }}
-      {{ if $menus.copyright_footer }}
+      {{ if .Site.Menus.legal_footer }}
```
Comment (stp-ip) [inline layouts/partials/fragments/copyright.html]: We are not quite consistent with the naming. The fragment is copyright. The translation key is legalFooter and the menu is legal_footer. Probably should be all moved to copyright or something more abstract.

File: exampleSite/config.toml (hunk 1 above)
Comment (mpourismaiel) [reply exampleSite/config.toml]: Fixed in 02991898802d2faa7c8be867619a655643a2edb9

File: exampleSite/config.toml (hunk 2 above)
Comment (mpourismaiel) [reply exampleSite/config.toml]: Fixed in 02991898802d2faa7c8be867619a655643a2edb9

File: layouts/partials/fragments/copyright.html (hunk 3 above)
Comment (mpourismaiel) [reply layouts/partials/fragments/copyright.html]: Fixed in 02991898802d2faa7c8be867619a655643a2edb9

File: exampleSite/config.toml (hunk 2 above)
Comment (mpourismaiel) [reply exampleSite/config.toml]: Fixed in 02991898802d2faa7c8be867619a655643a2edb9

**Pull request merged**